phabricatoremails/model/phabricator.py

```diff
@@ -0,0 +1,150 @@
+from dataclasses import dataclass
+from datetime import datetime
+from enum import Enum
+from typing import Optional, List, Union
+
+from phabricatoremails.model.common import Recipient, Reviewer
+
+
+@dataclass
+class Bug:
+    id: int
+    name: str
+    link: str
+
+
+@dataclass
+class Revision:
+    id: str
+    name: str
+    link: str
+    bug: Optional[Bug]
+
+
+@dataclass
+class ReplyContext:
+    other_author: str
+    other_date_utc: datetime
+    other_comment: str
+
+
+class DiffLineType(Enum):
```

Comment:
Do the values here matter? Are they being stored in a database? If not, I would use `auto`. If they are, I would suggest keeping them consistent with the variable names.

Reply:
These have the same situation as the [other enums](https://github.com/mozilla-conduit/phabricator-emails/pull/1#discussion_r403366214).

The consistent pattern I'm using here is:
* JSON keys are `camelCase` (due to matching PHP member variables, which is formatted according to PHP convention)
* JSON enum values are `kebab-case` (enum values need to match this)
* Enum keys are `CAPITAL_SNAKE_CASE` (due to python convention)
* Python variables are `snake_case` (due to python convention)

I _can_ change the JSON enum values, if you think it would have a positive impact.

Reply:
Rolling this discussion in with [this one](https://github.com/mozilla-conduit/phabricator-emails/pull/1#discussion_r403292804) and resolving